toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1566,6 +1567,12 @@ func ProvisionUserSSHCerts(installChroot safechroot.ChrootInterface, username st
 		return
 	}
 
+	// Avoiding warnings about uncaught overflow from code quality tooling.
```

Comment:
Not sure what this comment means

Reply:
Just explaining why we're adding two integers through a function instead of a regular `+`. I'm open to rewording suggestions, though.:)